src/demo-app/a11y/progress-spinner/progress-spinner-a11y.html

```diff
@@ -0,0 +1,16 @@
+<section>
+  <h2>Loading indicator (Indeterminate progress spinner)</h2>
+  <md-progress-spinner mode="indeterminate" color="primary" [strokeWidth]="1">
```

Comment:
Instead of `<md-progress-spinner mode="indeterminate">`, you can just do `<md-spinner>` (which is indeterminate by default)